lib/configuration/app_text.dart

```diff
@@ -0,0 +1,9 @@
+import 'package:flutter/material.dart';
+
+class AppText {
```

Comment:
same as in `AppColors`

Reply:
also better to have this app text class for storing text styles, not texts themselves. Better practice for it will be internatiolazion approach, like l10n or l18n which is default technology out from the box with flutter - https://docs.flutter.dev/development/accessibility-and-localization/internationalization

Reply:
fixed successfully